### ISSUE TITLE
Will vercel-php ever have gd library?

### ISSUE THREAD

**Issue by BionicBlueBell** [vercel-community/php#481]
My backend needs image compression and the most popular way to do it is with the gd library.

There is a workaround without using the gd library and that is by using the Imagick extension. The extension isn't readily available and the documentation does say something about being able to install the extension but I have no clue how to do it.

**Comment by Nxrix** [comment]
I have the same problem.
Fatal error: Uncaught Error: Call to undefined function imagecreatetruecolor() in /var/task/user/api/pixel8.php:76 Stack trace: #0 {main} thrown in /var/task/user/api/pixel8.php on line 76

**Comment by trikibilel** [comment]
Yeah i also need the gd extention to be present , it's a well known and popular extension to be added

**Comment by f3l1x** [comment]
Hi, I would really really love to support GD or ImageMagic, but it's really hard to do it, I am in dead end right now. So, I can tell you if or when `vercel-php` will support these extensions.
